### PR TITLE
Bug 1172465 - border-radius should be applied to all buttons

### PR DESCRIPTION
**Pull request by ayushagrawal288**
Corrected the CSS according to Alphabetical order.
And fixed one more button

### CODE REVIEW

**Comment by gurumukhi** [comment]
@ayushagrawal288 Travis checks failed, can you please check the error message and try to rectify them?

File: src/media/css/account.styl [hunk 1]
```diff
@@ -101,6 +101,7 @@
         margin: 30px 0 20px;
     }
     .button {
+    	border-radius: 5px;
```
Comment (gurumukhi) [inline src/media/css/account.styl]: Indentation is not proper, please given one more space before this line.

File: src/media/css/marketplace-elements.styl [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+        width: (2 / 3 * 100)%;
+    }
+    .mkt-prompt-btn-wrap {
+        justify-content: center;
+
+        button[type="submit"] {
+            max-width: $mobile-content-width;
+        }
+    }
+}
```
Comment (gurumukhi) [inline src/media/css/marketplace-elements.styl]: Are you sure you want this file to be added? Looks like these changes are not related to this bug.

File: src/media/css/newsletter.styl [hunk 3]
```diff
@@ -204,9 +204,10 @@ $arrow-size = 10px;
     }
     .newsletter-signup-button,
     .newsletter-cancel-button {
-        width: auto;
-        margin: 0;
         border-radius: 5px;
+        margin: 0;
+        width: auto;
+
```
Comment (gurumukhi) [inline src/media/css/newsletter.styl]: This extra empty line is not necessary.

File: src/media/css/marketplace-elements.styl (hunk 2 above)
Comment (ayushagrawal288) [inline src/media/css/marketplace-elements.styl]: Actually sir, this file is needed to add border-radius to Submit Feedback button

**Comment by ayushagrawal288** [comment]
@gurumukhi sir can u help me with how to solve the travis checks failed error?

**Comment by gurumukhi** [comment]
Adding the bug mentor @tsl143 to help @ayushagrawal288

**Comment by ayushagrawal288** [comment]
@tsl143 sir plz help me with travis checks failed error

**Comment by tsl143** [comment]
@ayushagrawal288, do we actually need to add complete marketplace-elements.styl file?

**Comment by ayushagrawal288** [comment]
@tsl143 sir found an replacement for marketplace-elements.styl file 
created an pr from a new branch: bug_1172465-border_radius
PR Request: https://github.com/mozilla/fireplace/pull/1467
Please Review

**Comment by gurumukhi** [comment]
Cool ! Someone from marketplace team will review that PR soon. You can close this PR.
